lib/v2/xiaote/index.js

```diff
@@ -0,0 +1,48 @@
+const got = require('@/utils/got'); // 自订的 got
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { data } = await got.post('https://lcen.xiaote.net//api/graphql/', {
+        json: {
+            query: `query($startCursor: Int) {
+                communities(startCursor: $startCursor) {
+                    edges {
+                        node {
+                            objectId
+                            content
+                            createdAt
+                            imageUrls
+                            user{
+                                nickname
+                            }
+                    }
+                }
+              }
+            }`,
+        },
+    });
+
+    ctx.state.data = {
+        title: '小特社区',
+        // 源链接
+        link: 'https://xiaote.com/',
+        // 遍历所有此前获取的数据
+        item: data.data.communities.edges.map((node) => {
+            const item = node.node;
+            let description = item.content;
+            if (item.imageUrls) {
+                item.imageUrls.forEach((url) => {
+                    description += `<img src="${url}">`;
+                });
+            }
+
+            return {
+                title: item.content.substring(0, Math.max(item.content.length, 12)) + '...',
```

Comment:
RSSHub will [trim title](https://github.com/DIYgod/RSSHub/blob/04820c33d006c2d1afb88c52b20b0561a4171086/lib/middleware/template.js#L70) if it's longer than `TITLE_LENGTH_LIMIT`.